IngridFuentes/frontendstudygroup.github.io
# Test resource Component

Test Cases

`More info` Link validation

- Check the `More info` link 12 times rendered on `/resources page` 
- Check when clicking `More info` link routes to corresponding page `/resources/resource.id` 
- Check each page has `resource.name` using getByText on the screen
